.github/workflows/ci.yml

```diff
@@ -40,6 +40,32 @@ jobs:
       - name: Run lint checks
         run: |
           mvn compiler:compile -Pdev,jdk11 -B -U -e
+  check-format:
```

Comment:
Can we do simply like `Run link checks` above and add the `Run format checks` as a separate step of `quick-build` instead of creating a new job?

Reply:
Oh yeah, I didn't notice those were separate steps.  I think it might be helpful to have the output separate though, since the build & tests & lint should be passing before approval, but format should be done after.  It's easier (for me at least) to see that w/ fully separate tasks.  And I don't know of any way to have multiple check results from the same job.

Reply:
But if your job fail, you can see pretty easily at which step the failure happened. I really see the format check as the same thing as the lint checks so I think we should replicate how it is done, at least for consistency? Also avoid preparing multiple times these containers makes the quick builder "quicker". Finally, since a `mvn install` is done in the `quick-build` prior to the checks, it will allow us to validate the format of the generated files as well (which we want, right?)

Reply:
Yeah, if you open the log and read through.  I still think it's easier to see at a glance (i.e. in the emails it sends you, or from the PR list) if it's in a separate job, and they should run in parallel.  The difference from the lint checks, imo, is that this is supposed to be done after approval, otherwise I'd make it part of the same job.  And that's going to be tricky enough for new contributors (backing out format changes is often hard).  

I haven't been formatting the generated code, it's not generated as complaint, so we'd need to essentially do a `spotless:apply` as part of generation.  It would affect the protobuf stuff too.  I can, but it didn't seem like something we really needed.

Reply:
> The difference from the lint checks, imo, is that this is supposed to be done after approval

That's true! Ok let's try it like this then